.mocharc.yml

```diff
@@ -0,0 +1,5 @@
+recursive: true
```

Comment:
does this duplicate the stuff in `package.json` (which also lists `require: `esm`, as well as `test/_setup` not listed here)

Reply:
it was required for `npm run test`, because `node_modules/.bin/mocha` runs node independently